ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -132,7 +132,7 @@ BEGIN
     IDS_EXP_SHEXT_L2            "this utility as shell extension using the option below."
     IDS_EXP_SHEXT_LEARN         "Learn more"
     IDS_EXP_SHEXT_REGISTER      "Register as shell extension"
-    IDS_EXP_DISABLECTXMENU      "Disable the Windows 11 context menu"
+    IDS_EXP_DISABLECTXMENU      "Disable the simplified Windows 11 context menu and activate the full Windows 10 menu"

```

Comment:
The "and activate the full Windows 10 menu" doesn't need to be there.

Reply:
Ok it's smoother